builtin/providers/aws/resource_aws_route_table.go

```diff
@@ -361,7 +361,10 @@ func resourceAwsRouteTableDelete(d *schema.ResourceData, meta interface{}) error
 func resourceAwsRouteTableHash(v interface{}) int {
 	var buf bytes.Buffer
 	m := v.(map[string]interface{})
-	buf.WriteString(fmt.Sprintf("%s-", m["cidr_block"].(string)))
+
+	if v, ok := m["cidr_block"]; ok {
```

Comment:
oh whoops - didn't change next line to use `v` - this would technically work but it's silly.

empty string is fine - you'll just generate a hash from the string w/ two dashes in it